### ISSUE TITLE
[1.20.X] Armor footstep accents with Presence Footsteps?

### ISSUE THREAD

**Issue by ArmoredFuzzball** [OreCruncher/DynamicSurroundingsFabric#55]
With the current Dynamic Surroundings version, if you install Presence Footsteps too, armor rattling noises when walking are gone, even with the footstep accents enabled (I assume because they are all deactivated internally). Presence Footsteps has no such armor noises, so it would be cool if they still worked!

**Comment by OreCruncher** [comment]
Armor accents should be playing. Only step related sounds affected by presence footstep is watery step, walking on leaves, and moving through brush.  The footstep sounds the mod plays is very small and should not be overriding anything presence footsteps is doing.

Which MC version and loader are you using?

**Comment by OreCruncher** [comment]
Yeah - I don't think I can work around this without making my life complicated or breaking PF. Basically, to suppress the normal footstep sound processing in Minecraft PF will fudge a data value to be artificially large. Because of this, the logic my footstep accent logic taps into will be bypassed.

**Comment by ArmoredFuzzball** [comment]
Sorry I was asleep. I'm using 1.20.2 (technically unsupported I know) with your 1.20.1 version of the mod, with fabric loader version 15.3.
I figured this may be a significant change that isn't worth it, so thanks for looking into it anyway.

**Comment by ArmoredFuzzball** [comment]
Guess I should take this to PF and see if they can make their mod more compatible

**Comment by OreCruncher** [comment]
You could, but I don't think there is anything they can do.  I did the same thing with my footstep sound acoustics way back when. It's a non-complex way to make Minecraft not play footstep sounds for an entity.  Modern Minecraft things are more complicated which makes it the best way.

**Comment by ArmoredFuzzball** [comment]
I see, thanks for the info then. Feel free to close this.

**Comment by OreCruncher** [comment]
I'll leave open for now.  Something may occur to me.

**Comment by DeshiDesu** [comment]
I definitely hear the armor accents. They are just replaced with presence footstep's chain footsteps.
PresenceFootsteps-1.9.4+1.20.1.jar
Forge 47.3.5

**Comment by DeshiDesu** [comment]
I was right, since DS uses chainmail equip sounds for the armor footstep accents its completely replaced by PF
![image](https://github.com/user-attachments/assets/88a3bb4b-fb76-4f30-af89-527520b1b3c3)

Not related but why use the equip sounds instead of the armor accents from the forge version? They were perfectly fine.

**Comment by ArmoredFuzzball** [comment]
Well, I'm using Fabric, so I'm not sure about forge. But I just retested on 1.20.1 with loader 16.0, and the issue remains, but maybe I can refine it now.
Presence Footsteps does have armor rattles when walking. They're very subtle, but I can hear them. I don't recall those being there before so maybe that was added between then and now. It does NOT however have chain armor rattles, just iron/gold/diamond/netherite.

If I hit the "disable all sounds" slider in Presence Footsteps, Dynamic Surrounding's armor rattles take over. In my opinion, this mod's rattles are much better, and it also handles chain armor as well.

PF has a setting to turn "Footwear" on and off, but even with it set to off, DS rattles do not work.

So it would be cool if they worked when setting Footwear to off in PF. Maybe this would evade the issue we discussed before with code collision?

**Comment by DeshiDesu** [comment]
You need to disable the PF built in texture pack since it overrides the sounds DS uses for their armor accents. And just a tip if you dont like the sounds being subtle you can adjust the volume in DS's assets and create your own resource pack to override the default settings.

PF can also be customized using resource packs so I managed to do something like this:

https://github.com/user-attachments/assets/3b2bb3e5-603e-47d4-93e8-05b66d135f67

**Comment by Steveaaaaa** [comment]
> You need to disable the PF built in texture pack since it overrides the sounds DS uses for their armor accents. And just a tip if you dont like the sounds being subtle you can adjust the volume in DS's assets and create your own resource pack to override the default settings.
> 
> PF can also be customized using resource packs so I managed to do something like this:
> 
>  Replay.2024-08-02.22-25-38.mp4

Could you explain how to do this? My footstep sound volume is too low I think:(

**Comment by ghost** [comment]
> You need to disable the PF built in texture pack since it overrides the sounds DS uses for their armor accents. And just a tip if you dont like the sounds being subtle you can adjust the volume in DS's assets and create your own resource pack to override the default settings.
> 
> PF can also be customized using resource packs so I managed to do something like this:
> 
>  Replay.2024-08-02.22-25-38.mp4

could you public a instance res_pack？

**Comment by BoringQuilt** [comment]
feel so good to know it's not only me who has faced this question
